app/models/concerns/spotlight/image_derivatives.rb

```diff
@@ -1,57 +1,26 @@
 module Spotlight
   ###
-  # Include Spotlight::ImageDerivatives in a class or module to get
-  # the derivative configurations in #spotlight_image_derivatives.
+  # Stores the derivative configurations in #spotlight_image_derivatives.
   # A new derivative could theoretically be added by putting the
   # following in an initializer.
   # Spotlight::ImageDerivatives.spotlight_image_derivatives << {
   #   version: :my_version,
-  #   field: :my_field,
-  #   lambda: lambda {|_|
-  #     version :my_version do
-  #       process :resize_to_fill => [30,30]
-  #     end
-  #   }
+  #   field: :my_field
   # }
-  #
-  # This will then create that new CarrierWave version in any class that extends this module
-  # and calls the apply_spotlight_image_derivative_versions class method described below.
   module ImageDerivatives
```

Comment:
There is definitely opportunity for refactoring, but I believe this is currently serving a purpose and can't quite be removed right now.  It provides the ability to configure the different solr fields that are used to populate the image references serialized into the sir-trevor block data.

I think when we take on #1673 (basically next) will be a good time to come up with a better way to configure which image versions are used in the application (I'll add a note in that ticket about doing so).